### PR TITLE
minor fix for the AMI recipe

### PR DESCRIPTION
**Pull request by JinZr**
related to issue #1177

### CODE REVIEW

File: lhotse/recipes/ami.py [hunk 1]
```diff
@@ -201,8 +201,12 @@ def download_audio(
             )
         elif mic == "mdm":
             for array in MDM_ARRAYS:
-                for channel in MDM_CHANNELS:
-                    wav_name = f"{item}.{array}-{channel}.wav"
+                if item == "IB4001" and array == "Array2":
```
Comment (desh2608) [inline lhotse/recipes/ami.py:204]: You could also just put a try/except block here and warn the user for failing downloads.

Comment (JinZr) [reply lhotse/recipes/ami.py]: good call! this would save a lot of effort.

File: lhotse/recipes/ami.py [hunk 2]
```diff
@@ -207,9 +207,14 @@ def download_audio(
...
                     wav_dir.mkdir(parents=True, exist_ok=True)
                     wav_path = wav_dir / wav_name
-                    resumable_download(
-                        wav_url, filename=wav_path, force_download=force_download
-                    )
+                    try:
+                        resumable_download(
+                            wav_url, filename=wav_path, force_download=force_download
+                        )
+                    except:
```
Comment (desh2608) [inline lhotse/recipes/ami.py:214]: PEP8 does not allow a bare `except` clause (see [this](https://stackoverflow.com/questions/14797375/should-i-always-specify-an-exception-type-in-except-statements/14797508#14797508) for why you should specify an exception type).

Comment (JinZr) [reply lhotse/recipes/ami.py]: ok thanks! the next commit should fix the issue

Comment (desh2608) [reply lhotse/recipes/ami.py]: Thanks. Is this ready to merge?

**Comment by JinZr** [comment]
i just ran through the process again, it is ready to be merged now.

thanks Desh!

**Review by desh2608 — APPROVED** [review]

**Comment by desh2608** [comment]
Thanks for the fix!

**Pull request merged**